meetings/2024-05-08.md

```diff
@@ -28,25 +40,102 @@
+* SC: Lastly, I will PR with some of the other documents listed under https://solidproject.org/TR/ along these lines.
+* IJ: We have the goal to ??? and avoid confusion with other documents. Two models. Model 1: any specification produced by a CG is subject to requirements for CG reports. A subclass is considered "not yet adopted" by the CG.. I see the use of ED, that might not be the best label. Benefit: IPR/authorship/ownership is clearly the CG and its participants. Model 2: these aren't CG reports until approved by the CG. Prior to that there areno markings/styling indicating that it's a CG report. Priot to that they should not be referred to as published CG reports. No W3C branding, just documents that will eventually be submitted. 
+* eP: Is the intention all current drafts are CG drafts? We have work item process that people can start repos but once moved to Solid org they become community drafts. In WG there is WD and ??? We could have something similar. there are impleentations following the draft at different levels/versions. 
+* IJ: Within CG reports there are two statuses: draft and final. Final is for improving patent policy coverage. If you have other labels you will have to educate the comunity on those. I have a slight preference for all being CG drafts until declared final. If these are all works of the CG they should be CG reports as opposed to external work. 
+* TBL: Solid GH org follows the Solid process. TR contains documents that are not necessarily ??? If we don't have the 'final' flag ??? 
+* IJ: I'd rather not entertain the proposal to map CG process to Solid Team? process. I don't want to change Solid process, just find a way to label it within the CG process. FINAL is optional, not required. Status within the CG system can be comunicated. 
+* eP: I think it is sufficent to have everything use the template. We can use semantic versioning and some kind of tag releases. About FINAL: keeping in mind there's a WG proposed, we plan to submit some of the CG drafts, not all. Some will continue to be within CG. Those should be tagged as FINAL? And when is it a good time in coordination with the WG? 
+* IJ: The two can be decoupled. If the charter is being reviewed, it should set expectations as to which deliverables will be picked up. The transfer happens whenever both parties are ready. FINAL is advised but not required. 
+* TBL: Is it FINAL when the WG picks it up?
+* IJ: When it hits the WG the policy is that every participant maes a full commitment over the text of the spec. Anyone in the WG that was also in the CG, it's not that relevant because they don't need to make the same commitment. It is relevant when someone has been in the CG but hasn't made commitment over the full text and doesn't join the WG..
```

Comment:
```suggestion
* IJ: When it hits the WG the policy is that every participant makes a full commitment over the text of the spec. Anyone in the WG that was also in the CG — it's not that relevant because they don't need to make the same commitment. It is relevant when someone has been in the CG but hasn't made commitment over the full text _and_ doesn't join the WG.
```